lib/models/entitiesStore.js

```diff
@@ -52,6 +52,9 @@ function findSilentEntities(service, subservice, ruleData, func, callback) {
         } catch (e) {
             return callback(e, null);
         }
+    } else {
+    	//Added default else clause
+        logger.debug('findSilentEntities() - Default else clause');
```

Comment:
Review indentation please (`//` is not aligend with `logger`)

Reply:
I checked the last commit and I can't locate the indentation error
https://github.com/fiqare-emergya-dev/cdti-fiqare-perseo-fe-src/blob/fiqare-improvements-1/lib/models/entitiesStore.js

Reply:
The diff shows the miss-indentation:

![imagen](https://user-images.githubusercontent.com/1534240/72162244-e01e2c80-33c1-11ea-8ca2-f293ce189444.png)

Maybe it's someking of tabs vs. space issue. Tabs shouldn't be used in Perseo FE code, according to our code style.

Reply:
Fixed in ff07d1070d5096cdbc0e359130c6a04514f87ea7

Reply:
Fixed in 0865dc81b89788cff47bcc99fb97d808534312bb